machineconfiguration/v1alpha1/0000_80_machineconfigstate.crd.yaml

```diff
@@ -0,0 +1,185 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: machineconfigstates.machineconfiguration.openshift.io
+  labels:
+    "openshift.io/operator-managed": ""
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
Please take a look at other TechPreview CRDs, there are a couple of annotations you need, one to tell the CVO that this is a tech preview feature, the other is the API approved annotation which should then have the value of this PR number

Reply:
I had the techprieview annotation but weirdly enough that broke `make update-codegen-crds` (it caused no changes to be made to the CRD when I updated the types) any idea why that is?

Reply:
You need to make sure the API group has a `.codegen.yaml`, see other examples where they've set up techpreview, eg https://github.com/openshift/api/blob/f404f2bc3524e03f3bc29a7b3cd0f624cd36eae7/config/.codegen.yaml#L1-L6